### ISSUE TITLE
16 bit images appear darker than they should be

### ISSUE THREAD

**Issue by Stoyan-I-A** [napari/napari#6534]
### 🐛 Bug Report

Here's a sreenshot of a 16 bit image loaded in napari (left) and the same image loaded in Windows' image viewer:
![image](https://github.com/napari/napari/assets/43790661/eebb90d0-8a0c-4c52-947b-1fd38a042c22)

The image appears much darker in the napari viewer than it should be and this makes some details of the image harder to see.

### 💡 Steps to Reproduce

1. Download the 16 bit image here: https://drive.google.com/file/d/1rNIvr-RtQnlidL1J3O96g_d1D22QsEW5/view?usp=sharing

(The file is to more than 10MB so GitHub won't let me attach it here).

2. Open napari
3. Go to File -> Open File(s) -> navigate to where the file is, select and load the image

### 💡 Expected Behavior

The image should be look similar to what is shown in the Windows image viewer (see screenshot above).

### 🌎 Environment

napari: 0.4.17
Platform: Windows-10-10.0.19045-SP0
Python: 3.8.16 | packaged by conda-forge | (default, Feb 1 2023, 15:53:35) [MSC v.1929 64 bit (AMD64)]
Qt: 5.15.6
PyQt5: 5.15.7
NumPy: 1.24.2
SciPy: 1.10.1
Dask: 2023.3.1
VisPy: 0.11.0
magicgui: 0.7.2
superqt: unknown
in-n-out: 0.1.7
app-model: 0.1.2
npe2: 0.6.2

OpenGL:
- GL version: 4.6.0 NVIDIA 545.92
- MAX_TEXTURE_SIZE: 32768

Screens:
- screen 1: resolution 1920x1080, scale 1.0
- screen 2: resolution 1920x1080, scale 1.0

### 💡 Additional Context

This behavior happens on other systems as well.

This was originally discovered in an issue on another GitHub project for which we created a plugin for napari. Link to the issue: https://github.com/axondeepseg/axondeepseg/issues/737

**Comment by jnahlers** [comment]
I tried opening that image in version `0.5.0a2.dev486+g4d60a7ce`. Even though the image contains both 0 pixels and 65536 pixels, the auto-contrast for some reason sets the range to 15623 - 54971. I don't know how the auto-contrast limits are supposed to be calculated, but this seems like a bug.

Note that if you right-click the contrast limits slider and click 'full range', you can pull the sliders to the full range (0 to 2^16). However, even then the image doesn't look like in the screenshot of the  Windows image viewer. I _don't_ think is a bug on Napari's side! We don't know what kind of curves Windows applies to the image. In fact, when I open the image in Windows Photos on my computer, it first looks like the full range image in the Napari canvas, and then a second later the greylevels somehow get automatically adjusted in the background and it looks like in the screenshot.

**Comment by Czaki** [comment]
I apologize for your incontinence. This bug is already reported in  #6453, #6183, #5608 and there is trying to solve them in #6216, but there is no consensus of proper solution.

**Comment by jni** [comment]
Yes I wouldn't hold up Windows Photos as the gold standard here, but it's indeed unexpected behaviour. 😂 

Re how they are calculated, yes it's a long-standing issue, see #5608 and #6183, and #6216 is an attempt at a fix that stalled — it's tricky when trying to generalise over large lazy arrays that could be remote. But we are working to fix this! 🙏

**Comment by jni** [comment]
Ach @Czaki jinx! 😂

**Comment by jni** [comment]
(btw @jnahlers I noticed you're at Monash! We should hang out! 😊)

**Comment by jni** [comment]
(probably better to take that chat to napari.zulipchat.com though 😅)

**Comment by jni** [comment]
I can get an approximation of the windows setting by going to full range, then setting the gamma to ~0.55.

**Comment by Stoyan-I-A** [comment]
Thank you for the responses!
